stdlib/Dates/src/periods.jl

```diff
@@ -196,6 +196,13 @@ struct CompoundPeriod <: AbstractTime
     end
 end
 
+"""
+    periods(::CompoundPeriod) -> Vector{Period}
```

Comment:
I think if it's a non-exported public API method then the docstring needs to be `Dates.periods`. Would be good to confirm with others, but IIRC that's the convention, otherwise this is just a docstring to an internal method.

Reply:
Done